src/app/content/utils.ts

```diff
@@ -137,3 +139,8 @@ export const loadPageContent = async(loader: ReturnType<AppServices['archiveLoad
   const page = await loader.page(pageId).load();
   return page.content;
 };
+
+export const getBookPipelineVersion = (book: Book): string => {
+  const url = BOOKS[book.id]?.archiveOverride || getArchiveUrl();
```

Comment:
Is there a reason to not use the book config loader to get the archive override as well?